app/Http/Controllers/ClassroomController.php

```diff
@@ -75,7 +79,8 @@ public function show(string $id)
      */
     public function edit(string $id)
     {
-        //
+        $class = Classroom::findorfail($id);
```

Comment:
`findOrFail` is like so, make sure to type it properly, even if it can work with your syntax